Assets/_Scripts/TwitchChatBroadcaster.cs

```diff
@@ -0,0 +1,76 @@
+using TwitchLib.Client.Models;
+using TwitchLib.Unity;
+using UnityEngine;
+using System.Collections.Generic;
+
+public class TwitchChatBroadcaster : Singleton<TwitchChatBroadcaster>
+{
+    public List<TwitchChatListener> listeners = new List<TwitchChatListener>();
```

Comment:
Can we look into using a HashSet for this? Our design requires us to frequently Add & Remove listeners.